src/template-analyzer.ts

```diff
@@ -221,9 +227,10 @@ export class TemplateAnalyzer {
    */
   public traverse(visitor: Partial<Visitor>): void {
     const visit = (
-      node: treeAdapter.Node,
+      node: treeAdapter.ParentNode | treeAdapter.Node,
```

Comment:
how come the type changed here? a visitor should accept any kind of node

Reply:
Left over from something I was trying. Will remove